package.json

```diff
@@ -81,7 +81,7 @@
     "window-fetch": "0.0.9",
     "window-selector": "0.0.5",
     "window-text-encoding": "0.0.2",
-    "window-worker": "0.0.97",
+    "window-worker": "0.0.98",
```

Comment:
Probably shouldn't include this in this PR. There is another PR where it's held up on multi-platform testing.

Reply:
yep sorry about that one